src/TraceEvent/Symbols/NativeSymbolModule.cs

```diff
@@ -26,6 +26,30 @@ namespace Microsoft.Diagnostics.Symbols
     /// </summary>
     public unsafe class NativeSymbolModule : ManagedSymbolModule
     {
+        /// <summary>
+        /// Retrieves the S_HEAPALLOCSITE information from the pdb as described here:
+        /// https://docs.microsoft.com/en-us/visualstudio/profiling/custom-native-etw-heap-events?view=vs-2017 
```

Comment:
Nit: Remove the "/en-us" part of the URL and the "?view=vs-2017" query param. The URL will still work and it will redirect to any localized version and future VS version as necessary.

Reply:
```suggestion
        /// https://docs.microsoft.com/visualstudio/profiling/custom-native-etw-heap-events 
```